core/src/nucleus/state.rs

```diff
@@ -58,6 +58,76 @@ impl NucleusState {
         }
     }
 
+    // Return WASM from ZomeFnCall request
+    pub fn get_fn_wasm(&self, fc: ZomeFnCall) -> Result<DnaWasm, ZomeFnResult> {
+        // Must have DNA
+        let dna = self.dna.as_ref();
+        if dna.is_none() {
+            return Err(ZomeFnResult::new(fc, Err(HolochainError::DnaMissing)));
+        }
+        let dna = dna.unwrap();
```

Comment:
`dna?` to avoid the unwrap here

Reply:
`?` doesn't work here since the function does not return an Error.
So I'm keeping the unwrap() unless you have another suggestion?